config/initializers/bulkrax.rb

```diff
@@ -32,7 +32,7 @@ def headers(term = nil)
       properties
     end
 
-    config.fill_in_blank_source_identifiers = ->(obj, index) { "#{obj.importerexporter.id}-#{index}" }
+    config.fill_in_blank_source_identifiers = ->(obj, type, index, key_count) { "#{type}-#{index}-#{key_count}" }
```

Comment:
is there a reason that `obj` was left as an argument although we aren't using it anymore?